src/test/javascript/spec/component/text-submission-assessment/text-feedback-conflicts.component.spec.ts

```diff
@@ -320,4 +335,24 @@ describe('TextFeedbackConflictsComponent', () => {
         textFeedbackConflictsHeaderComponent.onPrevConflict();
         expect(component.rightSubmission).toBe(conflictingSubmission);
     });
+
+    it('should go back if there is no conflict', () => {
+        const clickSpy = jest.spyOn(component, 'didClickedButtonNoConflict');
+        fixture.detectChanges();
+        const button = fixture.debugElement.query(By.css('button'));
+        button.triggerEventHandler('click', null);
```

Comment:
I'd add a safeguard here to make sure the element exists in the first place:
```suggestion
        const button = fixture.debugElement.query(By.css('button'));
        expect(button).not.toBe(null);
        button.triggerEventHandler('click', null);
```